frontend/src/components/common/header/navbar.jsx

```diff
@@ -72,24 +73,33 @@ class NavBar extends Component {
             <Navbar collapseOnSelect expand="lg">
               <Navbar.Toggle className="ml-3" onClick={this.toggle} />
               <Navbar.Collapse isOpen={this.state.isOpen} navbar>
-                <Nav className="mx-auto my-auto align-links">
-                  {/* resources  */}
+                <Nav className="mx-auto">
                   <NavDropdown
                     className="nav-link"
                     to="/resources"
                     title="Youth Resources"
                     id="nav-dropdown"
                   >
-                    <NavDropdown.Item eventKey="4.1">
-                      Hospitals
+                    <NavDropdown.Item>
+                      <HashLink eventKey="4.1" to="/resources#hospitals">
+                        Hospitals
+                      </HashLink>
                     </NavDropdown.Item>
-                    <NavDropdown.Item eventKey="4.2">
-                      Urgent care
+                    <NavDropdown.Item>
```

Comment:
these `<NavDropdown.Item>` need to add a the href too.

Otherwise when you click outside of the name it will not work 

Also lets add some styles for these so that they match the other drop-downs links